### ISSUE TITLE
Support @mixin-content when using JS Mixins

### ISSUE THREAD

**Issue by sandrina-p** [postcss/postcss-mixins#79]
```scss
@define-mixin isIOS {
    .isIOS & {
        @mixin-content;
    }
}

.foo {
    color: blue;

    @mixin isIOS {
        color: red;
    }
}
```
output:
```css
.foo { color: blue; }
.isIOS .foo { color: red; }
```

Now, with the Mixin written with JS, how do I access `@mixin-content`? Maybe an API called `mixinNode.content` could do that.

```js
module.exports = (mixinNode) => {
    return {
        '.isIOS &': {
            ...mixinNode.content
        }
    }
};
```

Notice that here I'm using `&` from `postcss-nested`

**Comment by ai** [comment]
Fixed 5e13cfb

**Comment by ai** [comment]
Released in 6.2
